ReactWindows/ReactNative.Shared/UIManager/Events/TouchEventTypeExtensions.cs

```diff
@@ -25,6 +25,8 @@ public static string GetJavaScriptEventName(this TouchEventType eventType)
                     return "topMouseOver";
                 case TouchEventType.Exited:
                     return "topMouseOut";
+                case TouchEventType.PointerMove:
+                    return "topMouseMoveCustom"; // Using a name different from "topMouseMove" since this one is treated by touch infra in an unexpected way
```

Comment:
>topMouseMoveCustom [](start = 28, length = 18)

I think I have a solution for this that is pretty straightforward, I just need to get sign-off on the upstream `ReactNativeRenderer` implementation to support a check for a property `moveOnly` on the pointer objects that prevents a console.error from occurring.

Reply:
I'll send a PR with the relevant change.